subprojects/file-watching/build.gradle.kts

```diff
@@ -26,6 +26,7 @@ dependencies {
     testImplementation(testFixtures(project(":snapshots")))
 
     testImplementation(libs.commonsIo)
+    testImplementation(libs.gradleEnterpriseTestAnnotation)
```

Comment:
Does it work as well if you add the dependency here: https://github.com/gradle/gradle/blob/fc502b86e2a0915d2e678c98ce3c89db75582494/build-logic/jvm/src/main/kotlin/gradlebuild.unittest-and-compile.gradle.kts#L125-L138

Reply:
Heh... probably, was just looking exactly a place like this.

Reply:
Moved up there, seems to be working